packs/aws/actions/lib/ec2parsers.py

```diff
@@ -35,6 +130,8 @@ def selector(self, output):
             return self.parseInstance(output)
         elif isinstance(output, boto.ec2.volume.Volume):
             return self.parseVolume(output)
+        elif isinstance(output, boto.ec2.blockdevicemapping.BlockDeviceType):
```

Comment:
Created issue https://github.com/StackStorm/st2contrib/issues/387 for tracking. Since marshalling and unmarshalling object properly is my pet-peeve I might take a crack at it :P